velox/connectors/hive/HiveConfig.cpp

```diff
@@ -213,6 +213,25 @@ uint64_t HiveConfig::orcWriterMinCompressionSize(const Config* session) const {
       config_->get<uint64_t>(kOrcWriterMinCompressionSize, 1024));
 }
 
+std::optional<uint8_t> HiveConfig::orcWriterCompressionLevel(
+    const Config* session) const {
+  auto sessionProp = session->get<uint8_t>(kOrcWriterCompressionLevelSession);
```

Comment:
It can be simplified as:
```
return session->get(xxx, config_->get())?
```

Reply:
I tried this initially, but unfortunately it doesn't work with std::optional/folly::Optional 

the session->get has to return an int8_t and expects the second argument to return an int8_t but the config_->get has to return a folly::Optional<int8_t> so they are incompatible